generators/importanceSampler.h

```diff
@@ -0,0 +1,103 @@
+#ifndef IMPORTANCESAMPLER_H
```

Comment:
add a brief description what this class does and how it interfaces with BAT
maybe a description of the layout of the parameter array could go here as well